.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and Publish
+
+on:
+  push:
+    tags:
+    - '*'
```

Comment:
Probably best to trigger the upload on GitHub releases, as in e.g. [here](https://github.com/ntessore/healpix/blob/c3d421ca8fc02d737cf973b69176eec913dd0e31/.github/workflows/cibuildwheel.yml#L10-L12).

Reply:
Done!